sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+- Added option `BUILD_CRYPTO` to support building azure-core without the Cryptographic namespace (no hashing). ON by default.
+- Updated `base64` implementation.
+- Updated `Uuid` implementation for Linux.
```

Comment:
The reader of the CL might wonder what the update was or why it was made, otherwise, it isn't useful.

```suggestion
- Updated `base64` implementation to remove external dependency.
- Updated `Uuid` implementation for Linux to remove external dependency.
```